docs/ELRS_RECEIVERS.md

```diff
@@ -4,7 +4,7 @@
 
 ## Supported ELRS Receivers ##
 
-As of April 2024, ELRS receivers with an ESP8285 MCU are supported by mLRS. The following targets are supported:
+As of April 2024, the following ELRS receiver targets are supported:
```

Comment:
maybe just plainly "The following ELRS receiver targets are supported" :)

Reply:
Done